### ISSUE TITLE
Upgrade to Ecto 3

### ISSUE THREAD

**Issue by indocomsoft** [source-academy/cadet#338]
Many dependencies requires Ecto 3, and overall the ecosystem is also moving to ecto 3.

**Comment by martin-henz** [comment]
Who can do this?

**Comment by jiachen247** [comment]
Looks like we cant use timex_ecto anymore.
https://github.com/bitwalker/timex_ecto/issues/80
https://elixirforum.com/t/how-to-upgrade-an-app-using-timex-ecto-to-phoenix-1-4-and-ecto-3-0/17676

**Comment by martin-henz** [comment]
timex is deprecated (currently warning us)

So someone needs migrate away from timex. Who can do this?

**Comment by jiachen247** [comment]
the issue has already been fixed by timex and merged into timex master. just waiting on them to release the new version on hex. In the meantime, we can just ignore the warnings.

Timex seems to be the defacto time library for elixir I suggest we stick to it!!
https://github.com/bitwalker/timex/pull/504
